data/json/recipes/food/dairy_products.json

```diff
@@ -37,5 +37,23 @@
     "autolearn": true,
     "qualities": [ { "id": "COOK", "level": 1 } ],
     "components": [ [ [ "milk_cream_jar", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "ghee",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DAIRY",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "charges": 1,
+    "time": "20 m",
+    "autolearn": true,
```

Comment:
I don't think most people know how to make ghee automatically. You might want to create an Indian food cook book later, but for now it could just go in one of the existing ones.

Reply:
The Indian food cookbook is an idea I'd been kicking around. Glad you think it's a good idea too

Reply:
So, aside from curry_meat, curry_veggy, ghee, and *maybe* flatbread, what other recipes could belong in an Indian cookbook?

Reply:
Naan should be its own, slightly fattier thing, needing a tandoor. It's pretty specialized.

Reply:
I’m not adding a tandoor yet 😆

But an oven-related cooking updoot is something I’m thinking about

Reply:
Roti and chapati would still translate well as flatbread though

Reply:
Ok, cookbook in. It only has 4 recipes, but that can be expanded later on. Now I need to add it to lists so it spawns

Reply:
It's in itemgroups now. How's it looking?